source/pool/pool.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2022 Meroxa, Inc.
```

Comment:
Not that it matters, just noticed it...
```suggestion
// Copyright © 2024 Meroxa, Inc.
```